src/Parser/Mage/Fire/Modules/Features/Abilities.js

```diff
@@ -123,40 +139,34 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.ICE_BLOCK,
         category: Abilities.SPELL_CATEGORIES.DEFENSIVE,
-        cooldown: 240,
```

Comment:
Wouldn't be better to add 
```jsx
castEfficiency: {
   suggestion: false,
}
```
maybe together with `importance: ISSUE_IMPORTANCE.MINOR,` instead of removing the cooldown?

Reply:
In a bunch of Mage things we'd been skipping showing CD entirely because didn't like seeing the maximum casts in the cast efficiency tab at all .... really should be a settable flag somewhere instead of leaving out the CD tho.

Reply:
Yea this like "you used your interupt 3% of the time" or "you didn't cast ice block at all" just looks bad with the bar and the % imo.

I wanted it to show the number of casts, but not the efficiency %. I'll see if I can find a way to add a setable flag, but this is exactly how frost has it setup.

Reply:
added a castEfficiency flag to disable castEfficiency and therefore hide the percentage, bar, and max casts.